routes/index.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express')
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const passport = require('passport');
+const Users = require('../models/userModel')
+
+const router = express.Router();
+
+/**
+ * Login route
+ */
+router.post('/login', async (req, res) => {
+  const { username, password } = req.body
+
+  const user = await Users.findOne({ username: username })
+  if (!user) return res.status(400).json({ success: false })
+
+  // Check if password is correct
+  const isMatch = await user.isValidPassword(password)
+  if (isMatch) {
+    const payload = {
+      id: user._id.toString(),
+      username: user.username
+    };
+
+    // Generate JWT token
+    jwt.sign(
+      payload,
+      process.env.JWT_SECRET,
+      { expiresIn: "1 day" },
+      (err, token) => {
+        res.json({
+          success: true,
+          token: 'Bearer ' + token
```

Comment:
Como o javascript é matreiro, é sempre melhor fazer interpolação de strings em vez de as somar. Assim conseguimos garantir que, mesmo que alguma coisa corra mal e as variáveis não sejam do tipo que estamos à espera, o resultado final é sempre uma string.
```suggestion
          token: `Bearer ${token}`
```

Reply:
fixed 👍